docs/README.md

```diff
@@ -0,0 +1,19 @@
+# Win32 App Isolation Introduction
+
+### Target Applications
+
+* Win32
+* Centennial
+
+### Minimum Requirements:
+
+* Windows insider version >= 25357
```

Comment:
Canary channel is the only one where these builds are currently available so we should probably specify that.